layouts/page/cuml-accel.html

```diff
@@ -0,0 +1,329 @@
+          </div>
+        </div>
+
+      <h1 class="mt-10 mb-4"> How It Works</h1>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Under the Hood </h2>
+          <p>When you load cuml.accel, importing scikit-learn (or umap-learn or hdbscan) allows cuML to "intercept"
+            estimators from these CPU modules. When you create a new estimator to run a machine learning algorithm, a 
+            "proxy estimator" is created instead.</p>
+
+          <code class="code-sample mt-3">
+            In [1]: %load_ext cuml.accel <br>
+            In [2]: from sklearn.cluster import KMeans <br><br>
+
+            In [3]: type(KMeans) <br>
+            Out[3]: cuml.internals.base_helpers.BaseMetaClass;
+          </code>
+
+          <br><br>
+
+          <p>Estimators that are implemented in cuML will be dispatched to run on the GPU where possible, and fall 
+            back to the CPU library otherwise. This applies to scikit-learn operations in your code, as well as third 
+            party libraries you may be using.</p>
+
+          <br>
+
+          <p>If a model is constructed on the GPU and then a method is called that is not implemented in cuML,
+            cuml.accel will reconstruct the model on the CPU and gracefully fall back to the equivalent scikit-learn
+            function instead.</p>
+
+        </div>
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-diagram-subtask"></i> Execution Flow </h2>
+          {{ with .Resources.GetMatch "chart_purple.png" }}
+          <img src="{{ .RelPermalink }}">
+          {{ end }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!-- Get Started -->
+<!-- <div id="get-started"></div> -->
+<div class="top-slant-container">
+  <div class="top-slant-up bg-rapids-gray"></div>
+</div>
+
+<section class="row section bg-rapids-gray padding-top-slant padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> Get Started with <br> Accelerated Machine Learning </h1>
+      <div class="row">
+        <div class="col py-3">
+          <h2><i class="fa-light fa-bookmark"></i> Try Now on Colab</h2>
+          {{ $colabImg := ((site.GetPage "_partners/google-colab").Resources.Get "logo.png").Resize "200x" }}
+          <a href="https://colab.research.google.com/github/rapidsai-community/showcase/blob/main/getting_started_tutorials/cuml_sklearn_colab_demo.ipynb" target="_blank"><img src="{{ $colabImg.RelPermalink }}"></a>
+          <p>Get started accelerating your machine learning workflows in a free GPU-enabled notebook environment using
+            your Google account. <a href="https://colab.research.google.com/github/rapidsai-community/showcase/blob/main/getting_started_tutorials/cuml_sklearn_colab_demo.ipynb" 
```

Comment:
I spot-checked this notebook and it worked for me. (I didn't run it all the way through.)